test/functional/test-dom.js

```diff
@@ -641,6 +641,20 @@ describes.sandboxed('DOM', {}, env => {
     });
   });
 
+  describe('getDataParamsFromLinkUrl', () => {
```

Comment:
Add tests if for the case where there is no href. If the URL is relative and so on.